### ISSUE TITLE
[Regression] Can't fabricate model with polymorphic belongs_to anymore

### ISSUE THREAD

**Issue by peret** [paulelliott/fabrication#295]
I have these models

    class Survey < ActiveRecord::Base
      belongs_to :surveyable, inverse_of: :survey, polymorphic: true
    end

    class Guestlist < ActiveRecord::Base
      has_one :survey, as: :surveyable
    end

and this Fabricator:

    Fabricator(:survey) do
      name { sequence(:name) { |i| "Survey No. #{i}" } }
      surveyable fabricator: :guestlist
    end

This used to work, up until version 2.14.1 of this gem. Since 2.15.0, calling `Fabricate(:survey)` throws a `NameError: uninitialized constant Survey::Surveyable`. All subsequent calls to the Fabricator then seem to return `nil`.

**Comment by peret** [comment]
Any progress on this issue? I can't upgrade the gem because of this.

**Comment by paulelliott** [comment]
Yeah, I think it was introduced by 7c445d8bea236e6fea4e90f57fc8f934ac68b85f. Could you see if the `polymorphic-belongs-to-fix` branch for PR #298 works for you? I just reverted that commit to validate it is actually what is causing the issue. If it is I'll come up with an actual fix and cut a release.

```
gem 'fabrication', github: 'paulelliott/fabrication', branch: 'polymorphic-belongs-to-fix'
```

**Comment by peret** [comment]
Yes, using the `polymorphic-belongs-to-fix` does indeed fix it for me.
